Shopify/dawn
# Fix background gradient for Related Products

### PR Summary: 

Fixes the appearance of the background gradient for the Related Products section, when scroll animations are turned on. 

### What approach did you take?

We were previously animating the entire section (including the background). This doesn't match the way we're animating other sections. Generally, we should let the background be static, and only animate in the content of that section. 

This PR shifts to that approach — it moves the animation class to the direct child of the section instead. 

### Visual impact on existing themes

Before, when the scroll animation is turned on, the page's background gradient would break at the Related products section. With this PR applied, it won't.

Before|After
---|---
 <img width="1161" alt="Screenshot 2023-05-15 at 12 02 53 PM" src="https://github.com/Shopify/dawn/assets/1202812/9f7c2bf5-ffa2-4da6-8578-5729d53b82a1">|<img width="1161" alt="Screenshot 2023-05-15 at 12 09 45 PM" src="https://github.com/Shopify/dawn/assets/1202812/2598b22e-9c18-4420-a571-1f1f3003847e">



### Testing steps/scenarios

1. Visit the demo store's product page.
2. Make sure Theme Settings > Animation > "Reveal sections on scroll" is turned on. 
3. In Theme Settings > Colors, ensure there's a background gradient assigned to Scheme 1. 
4. Ensure that the Related Products section's background blends in fluidly with the page background. 

### Demo links


- [Editor](https://os2-demo.myshopify.com/admin/themes/139947376662/editor)

### Checklist
